ui/raidboss/data/05-shb/trial/emerald_weapon.txt

```diff
@@ -6,11 +6,11 @@
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+# Can use InCombat line to start timeline since the multiple phases do not
+# change the player's InCombat status.
```

Comment:
> There is a checkpoint in normal mode as well, and so this functions kind of like a door boss.

There is a checkpoint, but when I ran the fight, there were no 261 lines between phases (i.e. never out of combat), so that shouldn't cause a timeline reset, right?  Although now that I think about it, if there were a wipe after the checkpoint, that presumably would change the InCombat status, and the player would restart at the checkpoint, meaning the 0x104 sync would be off.  Hmm.  Okay, I'll revert these.